ticdc/manage-ticdc.md

```diff
@@ -220,6 +220,24 @@ The following are descriptions of parameters that can be configured for the sink
 
 For more parameters of Pulsar, see [pulsar-client-go ClientOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ClientOptions) and [pulsar-client-go ProducerOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ProducerOptions).
 
+#### Configure sink URI with cdclog
+
+The cdclog files can be used together with `br` to provide point-in-time (PiTR) recovery. This is experimental feature.
```

Comment:
Is the "cdclog file" here referring to "Files written on the local filesystem or on Amazon S3 compatible storage"?

Reply:
`cdclog` are files written by TiCDC. CDC stands for change data capture.

Reply:
```suggestion
The `cdclog` files (files written by TiCDC) can be used together with Backup & Restore (BR) to provide point-in-time (PiTR) recovery. This is an experimental feature.
```